helpers/manifest.json

```diff
@@ -924,5 +924,108 @@
         "max": "50"
       }
     ]
+  },
+  {
+    "name": "Adjust colors",
+    "author": "minnieo",
+    "version": "0.1.0",
+    "label": "Adjust colors",
+    "login": false,
+    "recurs": true,
+    "desc": "Adjust the colors of the site.",
+    "entrypoint": "adjustSite",
+    "namespace": "adjust",
+    "page": "accessibility",
+    "catch_reset": [
+      "sepia",
+      "hueRotate",
+      "bright",
+      "saturate",
+      "contrast",
+      "upvote",
+      "downvote",
+      "boost"
+    ],
+    "fields": [
+      {
+        "type": "reset",
+        "key": "resetAdjust",
+        "label": "Reset values to defaults:",
+        "value": "reset"
+      },
+      {
+        "type": "range",
+        "initial": 0,
+        "key": "sepia",
+        "label": "Sepia",
+        "step": 0.1,
+        "min": 0,
+        "max": 10,
+        "show_value": true
+      },
+      {
+        "type": "range",
+        "initial": 0,
+        "key": "hueRotate",
+        "label": "Hue",
+        "step": 1,
+        "min": 0,
+        "max": 36,
+        "show_value": true
+      },
+      {
+        "type": "range",
+        "initial": 0,
+        "key": "bright",
+        "label": "Brightness",
+        "step": 0.1,
+        "min": 0,
+        "max": 10,
+        "show_value": true
+      },
+      {
+        "type": "range",
+        "initial": 0,
+        "key": "saturate",
+        "label": "Saturation",
+        "step": 0.1,
+        "min": 0,
+        "max": 10,
+        "show_value": true
+      },
+      {
+        "type": "range",
+        "initial": 0,
+        "key": "contrast",
+        "label": "Contrast",
+        "step": "0.1",
+        "min": "0",
+        "max": "10",
+        "show_value": true
+      },
+      {
+        "type": "color",
+        "initial": "#24B270",
```

Comment:
I was wondering where these default values come from--can we start by inheriting values from kbin, e.g., `var(--kbin-vote-text-color)`?